integrated_channels/canvas/models.py

```diff
@@ -133,3 +134,9 @@ def channel_code():
         Returns an capitalized identifier for this channel class, unique among subclasses.
         """
         return 'CANVAS'
+
+    def get_content_metadata_exporter(self, user):
```

Comment:
ah this reminds me I need to add a similar method on my branch to return the custom transmitter too!

Reply:
Yeah I only realized it when my exporter was not being hit _at all_